<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UnpinnDialog</class>
 <widget class="QDialog" name="UnpinnDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Unpinn</string>
  </property>
  <property name="windowIcon">
   <iconset theme="edit-cut">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0" colspan="2">
    <widget class="QCheckBox" name="Delete">
     <property name="toolTip">
      <string>If enabled items unpinned from object geometry will be deleted from current item list.</string>
     </property>
     <property name="text">
      <string>Delete unpinned objects</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="Class">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>&lt;p&gt;Classes of items to umpin from object geometry. Next values can be separated by comma sign.&lt;/p&gt;</string>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="classLabel">
     <property name="text">
      <string>Class</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="procedButton">
       <property name="text">
        <string>Unpinn</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QProgressBar" name="progressBar"/>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QCheckBox" name="Keep">
     <property name="toolTip">
      <string>If elabled keeps first and last relations.</string>
     </property>
     <property name="text">
      <string>Keep boundary relations</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>procedButton</sender>
   <signal>clicked()</signal>
   <receiver>UnpinnDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>350</x>
     <y>281</y>
    </hint>
    <hint type="destinationlabel">
     <x>352</x>
     <y>371</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>UnpinnDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>285</x>
     <y>281</y>
    </hint>
    <hint type="destinationlabel">
     <x>301</x>
     <y>420</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
